traits/stubs_tests/examples/Any.py

```diff
@@ -19,23 +19,23 @@ class Test(HasTraits):
 obj.i = "5"
 obj.i = 5
 obj.i = 5.5
-obj.var = {"a": 5, "b": 6}
-obj.var = {"a": 5.5, "b": 6}
-obj.var = {"a": 5, "b": None, "c": ""}
-obj.var = []
-obj.var = [1, 2, 3]
-obj.var = [1.1]
-obj.var = [1.1, 2, 3.3]
-obj.var = ''
-obj.var = "5"
-obj.var = 5
-obj.var = False
-obj.var = 5.5
-obj.var = 5 + 4j
-obj.var = True
-obj.var = [1, 2, "3"]
-obj.var = None
-obj.var = ['1']
+obj.i = {"a": 5, "b": 6}
```

Comment:
I don't know whether these `.var` assignments were intentional, but I'd be happy to take a leaf out of NumPy's book and have our initial stubs support _recommended_ use-cases instead of all possible use-cases. In this particular case, most code should be using `HasStrictTraits` rather than `HasTraits`.